client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -660,6 +662,14 @@ export default function CompositeCheckout( {
 		reduxDispatch( infoNotice( translate( 'Redirecting to payment partner…' ) ) );
 	}, [ reduxDispatch, translate ] );
 
+	const cartHasSearchProduct = useMemo(
+		() =>
+			items.some( ( { type } ) =>
```

Comment:
This is not a big deal, but I think it would be better to use `responseCart.products` here rather than `items`. The reason is that `items` is a type of data has been deprecated (see https://github.com/Automattic/wp-calypso/pull/49580) and should probably be removed (in fact, thanks for the reminder!).

Reply:
Awesome, makes sense. Thanks for catching that. 👍 
Fixed.